docs/tutorials/using-ev3-speaker.md

```diff
@@ -0,0 +1,196 @@
+
+Documentation:
+* [man page](https://manpages.debian.org/jessie/alsa-utils/amixer.1.en.html)
+
+Example:
+
+```bash
+$ amixer get Playback,0 # get current volume
+Simple mixer control 'Playback',0
+  Capabilities: volume volume-joined penum
+  Playback channels: Mono
+  Capture channels: Mono
+  Limits: 0 - 256
+  Mono: 255 [100%]
+$ amixer set Playback,0 50% # set volume to 50%
+Simple mixer control 'Playback',0
+  Capabilities: volume volume-joined penum
+  Playback channels: Mono
+  Capture channels: Mono
+  Limits: 0 - 256
+  Mono: 128 [50%]
+```
+
+### alsamixer Command
+
+Graphical volume controls.
+
+```
+┌───────────────────────────── AlsaMixer v1.0.25 ──────────────────────────────┐
+│ Card: LEGO Mindstorms EV3 speaker                    F1:  Help               │
+│ Chip:                                                F2:  System information │
+│ View: F3:[Playback] F4: Capture  F5: All             F6:  Select sound card  │
+│ Item: Playback                                       Esc: Exit               │
+│                                                                              │
+│                                     ┌──┐                                     │
+│                                     │  │                                     │
+│                                     │  │                                     │
+│                                     │  │                                     │
+│                                     │  │                                     │
+│                                     │  │                                     │
+│                                     │  │                                     │
+│                                     │  │                                     │
+│                                     │▒▒│                                     │
+│                                     │▒▒│                                     │
+│                                     │▒▒│                                     │
+│                                     │▒▒│                                     │
+│                                     │▒▒│                                     │
+│                                     └──┘                                     │
+│                                      40                                      │
+│                                  <Playback>                                  │
+│                                                                              │
+└──────────────────────────────────────────────────────────────────────────────┘
+```
+
+
+## More Fun!
+
+### Text-To-Speech
+
+```bash
+$ espeak "hello, I am an EV3.
+> I like to talk because I am a robot.
+> Did you know that robots like to make sounds?
+> Beep. Boop. Dit. Dit. Meep.
+> I am just such a chatterbox." --stdout | aplay
+Playing WAVE 'stdin' : Signed 16 bit Little Endian, Rate 22050 Hz, Mono
+```
+
+Tip: Add a function like this to your ```~/.bashrc``` file.
```

Comment:
I think this might be easier to follow if it had its own heading or were in a panel.